book/chrome.md

```diff
@@ -656,41 +681,49 @@ Each tab needs to make sure not to draw to those pixels:
 class Tab:
     def draw(self, canvas):
         for cmd in self.display_list:
-            if cmd.top > self.scroll + HEIGHT - CHROME_PX: continue
+            if cmd.top > self.scroll + HEIGHT - self.chrome_bottom:
+                continue
             if cmd.bottom < self.scroll: continue
-            cmd.execute(self.scroll - CHROME_PX, canvas)
+            cmd.execute(self.scroll - self.chrome_bottom, canvas)
 ```
 
-There are still sometimes going to be halves of letters that stick out
+Now let's turn our attention to designing the UI.
+
+First, there are still sometimes going to be halves of letters that stick out
 into the browser chrome, but we can hide them by just drawing over
-them:
+them (here I'm assuming we've already computed `self.bottom` (representing
+the bottom y coordinate of the browser chrome); that will
+come in a moment):
```

Comment:
Let's just move the outer parenthetical to a sentence, I think you're not supposed to do nested parentheses in normal prose :)